Makefile

```diff
@@ -23,7 +31,7 @@ apply:
 
 ## Runs govet and gofmt against all packages.
 .PHONY: check-style
-check-style: webapp/node_modules gofmt govet errcheck
+check-style: webapp/.npminstall gofmt govet errcheck
```

Comment:
"Normally" `golint` would be part of this target. But given that the code isn't compliant (jet) I leaved this out.

Reply:
Well done the add of golint! I have missed the addition in default plugins workflow but I :+1: now!

Reply:
There are currently 57 error when running `golint`. Fixing them will take some time :wink: